docs/github_actions.rst

```diff
@@ -0,0 +1,238 @@
+You should be able to watch the status of this workflow in the repository's Actions tab:
+
+.. image:: images/github_workflow.png
+   :alt: Screenshot showing a running Github Action workflow
+
+If you open a pull request for a branch that includes the workflow, you will find a section at the bottom of the pull request that shows the results of the checks that were performed by the workflow:
+
+.. image:: images/github_checks.png
+   :alt: Screenshot showing a successful check on a GitHub pull request
+
+It is possible to configure the repository's main branch as a *protected branch* so that changes can only be merged to it if these checks are passing.
+
+See GitHub's documentation for instructions to `configure protected branches <https://help.github.com/en/github/administering-a-repository/configuring-protected-branches>`_ and `enable required status checks <https://help.github.com/en/github/administering-a-repository/enabling-required-status-checks>`_.
+
+
+
+Run Headless Browser Tests
+--------------------------
+It is possible to run Robot Framework tests that control a real browser
+as long as the CI environment has the necessary software installed. For
+Chrome, it must have Chrome and chromedriver. For Firefox, it must have
+Firefox and geckodriver.
+
+Fortunately GitHub Actions comes preconfigured with an image that
+includes these browsers. However it is necessary to run the browser in
+headless mode. When using CumulusCIâ€™s ``robot`` task, this can be done
+by passing the ``-o vars BROWSER:headlesschrome`` option.
+
+Here is a complete workflow to run Robot Framework tests for any commit:
+
+.. code-block:: yaml
+
+   name: Robot Tests
+
+   on: [push]
+
+   env:
+     CUMULUSCI_KEYCHAIN_CLASS: cumulusci.core.keychain.EnvironmentProjectKeychain
+     CUMULUSCI_SERVICE_github: ${{ secrets.CUMULUSCI_SERVICE_github }}
+
+   jobs:
+     unit_tests:
+       name: "Run Robot Framework tests"
+       runs-on: ubuntu-latest
+       steps:
+       - uses: actions/checkout@v2
+       - name: Install sfdx
+         run: |
+           mkdir sfdx
+           wget -qO- https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz | tar xJ -C sfdx --strip-components 1
+           ./sfdx/install
+           echo ${{ secrets.SFDX_AUTH_URL }} > sfdx_auth
+           sfdx force:auth:sfdxurl:store -f sfdx_auth -d
+       - name: Set up Python
+         uses: actions/setup-python@v1
+         with:
+           python-version: "3.8"
+       - name: Install CumulusCI
+         run: |
+           python -m pip install -U pip
+           pip install cumulusci
+       - run: |
+           cci task run robot --org dev -o vars BROWSER:headlesschrome
+       - name: Store robot results
+         uses: actions/upload-artifact@v1
+         with:
+           name: robot
+           path: robot/CumulusCI-Test/results
+       - name: Delete scratch org
+         if: always()
+         run: |
+           cci org scratch_delete dev
+
```

Comment:
We've had a few questions recently about how to connect CumulusCI to a persistent org in the context of a CI system, which is possible but currently not documented. We should cover this as well.

Reply:
Should we also cover 

1. Building managed package versions
2. Deploying to a persistent production org

so that we have a complete example CI and implementation of CumulusCI Flow to reference?